tests/unit/archive_base_test.py

```diff
@@ -405,6 +405,407 @@ def test_check_missing_files(datapackage, baseline_resources, new_resources, suc
+                    mediatype="",
+                    format="",
+                    bytes=9,
+                    hash="",
+                ),
+            ],
+            True,
+        ),
+        (
+            # Test comparison ignores deleted files
+            [
+                Resource(
+                    name="resource0",
+                    path="https://www.example.com",
+                    remote_url="https://www.example.com",
+                    title="",
+                    parts={},
+                    mediatype="",
+                    format="",
+                    bytes=10,
+                    hash="",
+                ),
+                Resource(
+                    name="resource1",
+                    path="https://www.example.com",
+                    remote_url="https://www.example.com",
+                    title="",
+                    parts={},
+                    mediatype="",
+                    format="",
+                    bytes=10,
+                    hash="",
+                ),
+            ],
+            [
+                Resource(
+                    name="resource0",
+                    path="https://www.example.com",
+                    remote_url="https://www.example.com",
+                    title="",
+                    parts={},
+                    mediatype="",
+                    format="",
+                    bytes=10,
+                    hash="",
+                ),
+            ],
+            True,
+        ),
+        (
+            # Test comparison ignores added files
+            [
+                Resource(
+                    name="resource0",
+                    path="https://www.example.com",
+                    remote_url="https://www.example.com",
+                    title="",
+                    parts={},
+                    mediatype="",
+                    format="",
+                    bytes=10,
+                    hash="",
+                ),
+            ],
+            [
+                Resource(
+                    name="resource0",
+                    path="https://www.example.com",
+                    remote_url="https://www.example.com",
+                    title="",
+                    parts={},
+                    mediatype="",
+                    format="",
+                    bytes=10,
+                    hash="",
+                ),
+                Resource(
+                    name="resource1",
+                    path="https://www.example.com",
+                    remote_url="https://www.example.com",
+                    title="",
+                    parts={},
+                    mediatype="",
+                    format="",
+                    bytes=10,
+                    hash="",
+                ),
+            ],
+            True,
+        ),
+    ],
+)
+def test_check_file_size(datapackage, baseline_resources, new_resources, success):
+    """Test the ``_check_missing_files`` validation test."""
```

Comment:
I think you don't need this docstring at all, we should have disabled the linting requirement in test. But if you want to keep the docstring, this (and the below docstring) still refers to `_check_missing_files`.

Reply:
I think it's helpful for user legibility, but good catch! I've just updated them